install_config/registry/extended_registry_configuration.adoc

```diff
@@ -581,6 +586,35 @@ be safe to enable its support in the registry. See the
 xref:docker-registry-configuration-reference-middleware[middleware
 configuration reference] above for particular option.
 
+[[docker-registry-configuration-reference-openshift]]
+=== OpenShift
+
+This section allows to configure global settings related to {product-title}
+specific features. It will obsolete all the `*openshift*` related settings in
+label:docker-registry-configuration-reference-middleware[middleware section] in
+the future. At the moment, it allows to configure metrics collection.
+
+====
+----
+openshift:
+  version: 1.0 <1>
+  metrics:
+    enabled: false <2>
+    secret: PUT_YOUR_SECRET_HERE <3>
+----
+<1> Is a mandatory entry specifying configuration version of this section. The
+only supported value is `1.0`.
+<2> Can be set to `*true*` to enable metrics collection. The same can be
+achieved using the boolean environment variable
+`*REGISTRY_OPENSHIFT_METRICS_ENABLED=true*`.
+<3> A secret used to authorize client requests. Metrics clients must use it as
+a bearer token in `*Authorization*` header. The can be overridden with the
```

Comment:
`Authorization` header. It can be overridden by the environment variable `REGISTRY_OPENSHIFT_METRICS_SECRET`.